src/sdk/error/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -25,6 +25,9 @@ class ErrorBoundary extends Component {
     }
   }
 
+  // We can't accurately type the error here, since it could vary depending on
+  // what caused it. This is not standardized.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public componentDidCatch(error: any, errorInfo: ErrorInfo) {
```

Comment:
Does this error extend the Error? If yes, you can put type it there.

Reply:
Not necessarily :(
I could not find a reliable way to type this